src/modules/TaskGenerate.js

```diff
@@ -0,0 +1,50 @@
+import tasks from './tasks.js';
+// import trashIconPng from '../assets/trash-icon.png';
```

Comment:
it will seems better to remove comments 
because its not JavaScript best practice